src/overlays/actors/ovl_Bg_Tobira01/z_bg_tobira01.h

```diff
@@ -5,9 +5,17 @@
 
 struct BgTobira01;
 
+typedef void (*TobiraActionFunction)(struct BgTobira01*, PlayState*);
+
 typedef struct BgTobira01 {
-    /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x2C];
+    /* 0x000 */ DynaPolyActor dyna;
+    /* 0x15C */ TobiraActionFunction actionFunction;
```

Comment:
```suggestion
    /* 0x15C */ BgTobira01ActionFunc actionFunc;
```
This would be the abbreviations we use for consistency. `Func` is shortened and you want the full name in the action func typedef